worker/worker-manager/score-worker-manager-api/src/main/java/org/openscore/worker/management/monitor/WorkerMonitors.java

```diff
@@ -0,0 +1,10 @@
+package org.openscore.worker.management.monitor;
+
+import org.openscore.worker.management.services.WorkerMonitorInfoEnum;
+
+import java.io.Serializable;
+import java.util.HashMap;
+
+public interface WorkerMonitors {
+    HashMap<WorkerMonitorInfoEnum, Serializable> getMonitorInfo();

```

Comment:
I would really prefer a Map instead of a HashMap
although it means casting when throwing the event.